Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -158,6 +161,7 @@ void Preset(bool _bNTSC)
 
 	s_ticks_last_line_start = 0;
 	s_half_line_count = 1;
+	s_half_line_of_next_si_poll = (7 * 2) - 1; // first results at start of 7th line
```

Comment:
Would there be a useful constant name for `(7 * 2) - 1`, since it's used multiple times in here?